packages/material-ui-lab/src/Rating/Rating.d.ts

```diff
@@ -97,8 +97,8 @@ export interface RatingProps
   max?: number;
   /**
    * The name attribute of the radio `input` elements.
-   * If `readOnly` is false, the prop is required,
-   * this input name`should be unique within the parent form.
+   * This input `name`should be unique within the page.
+   * Being unique within a form is insufficient since the `name` is used to generated IDs.
```

Comment:
Does it make the name required even with read-only?

- If it does, I think that it should prevent the merge of the pull request. I don't think that it's OK to force setting a unique name to display a rating value.
- If it doesn't, all good from my end.

Reply:
The name is currently not required since we use `useId` as a fallback. The change regarding the "required-ness" is unrelated to this PR.